packages/manager/modules/pci/src/projects/project/public-gateways/delete/delete.routing.js

```diff
@@ -0,0 +1,20 @@
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state('pci.projects.project.public-gateways.delete', {
+    url: '/delete',
+    params: { gateway: null },
+    views: {
+      modal: {
+        component: 'pciProjectPublicGatewaysDelete',
+      },
+    },
+    resolve: {
+      goBack: /* @ngInject */ (goToPublicGateway) => goToPublicGateway,
+      gateway: /* @ngInject */ ($transition$) => $transition$.params().gateway,
+      breadcrumb: () => null,
+    },
+    layout: 'modal',
+    atInternet: {
+      ignore: true,
```

Comment:
```suggestion
      rename: 'public-gateway::delete',
```

Reply:
updated as suggested, moved all the page tracking inside routing.